src/ngraph/runtime/interpreter/int_executable.hpp

```diff
@@ -468,6 +468,25 @@ class INTERPRETER_BACKEND_API ngraph::runtime::interpreter::INTExecutable : publ
                                     broadcast_axes);
             break;
         }
+        case OP_TYPEID::Broadcast_v3:
```

Comment:
You don't want this; the interpreter tries evaluate first and only goes to the switch if evaluate returns false.

Reply:
ok, thanks. I will remove it before I do final push. I wanted to test if Broadcast v3, BIDIRECTIONAL is functional with current kernel. And my test works fine.